astropy/tests/pytest_repeat.py

```diff
@@ -0,0 +1,27 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+These plugins modify the behavior of py.test and are meant to be imported
+into conftest.py in the root directory.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+from ..extern.six.moves import range
+
+
+def pytest_addoption(parser):
+
+    parser.addoption('--repeat', action='store',
```

Comment:
This is technically a "new feature" in the 2.0.x series, right? I think that should be mentioned in the changelog.  I think it's justifiable in a 2.0.x release given that it's really tied to the testing architecture, but it should be mentioned rather than silently appearing.

Reply:
This is actually available in `v2.0.2`, so I don't think any action is required:
https://github.com/astropy/astropy/blob/93f7e758ee3113f83a150cc98ce52c1eb894b16e/astropy/tests/pytest_plugins.py#L112-L113

Reply:
Oh my mistake. all good then